config/k8s/provider-deployment-runtime-config.yaml

```diff
@@ -0,0 +1,16 @@
+---
+  apiVersion: pkg.crossplane.io/v1beta1
+  kind: DeploymentRuntimeConfig
+  metadata:
+    name: provider-kubernetes-runtime-config
+  spec:
+    deploymentTemplate:
+      spec:
+        selector: {}
+        template:
+          spec:
+            containers:
+              - name: package-runtime
+                args:
+                  - '--sanitize-secrets'
+                  - '--debug' 
```

Comment:
We probably shouldn't enable debug by default. Let's omit it for now.

Reply:
Updated.